geomesa-spark/geomesa-spark-sql/src/main/scala/org/apache/spark/sql/SQLGeometricConstructorFunctions.scala

```diff
@@ -0,0 +1,65 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.apache.spark.sql
+
+import com.vividsolutions.jts.geom._
+import org.geotools.geometry.jts.JTS
+import org.locationtech.geomesa.utils.geohash.{BoundingBox, GeoHash}
+import org.locationtech.geomesa.utils.text.{WKBUtils, WKTUtils}
+import org.apache.spark.sql.SQLContext
+
+object SQLGeometricConstructorFunctions {
```

Comment:
@Afterglow375 I agree that we ought not link to PostGIS's documentation. 

@mdzimmerman ideas for documenting the functions easily?